src/main/resources/hibernate.cfg.xml

```diff
@@ -5,14 +5,12 @@
 
 <hibernate-configuration>
     <session-factory>
-        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
-        <property name="connection.url">jdbc:mysql://localhost/cinema?serverTimezone=UTC</property>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL8Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/htwo?serverTimezone=UTC</property>
         <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
-        <property name="connection.username">ma_admin</property>
-        <property name="connection.password">1234</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">root</property>
         <property name="show_sql">true</property>
-        <property name="hbm2ddl.auto">update</property>
-
-
+        <property name="hbm2ddl.auto">create</property>
     </session-factory>
 </hibernate-configuration>
```

Comment:
Don’t change the properties in hibernate.cfg.xml for connection to database. You can do that only temporary for testing with your own schema;